terraform/environments/data-platform/tests/delete_table.py

```diff
@@ -0,0 +1,38 @@
+import json
```

Comment:
Is this test triggered from the workflow?

Wondering if this will fix the issue @murdo-moj raised here https://mojdt.slack.com/archives/C05AABSKFLJ/p1698325742203439

Reply:
These tests are for running locally and aren't triggered by the workflow.